### DISCUSSION TITLE
Why are old Rev.C1 AI decks still being shipped?

### DISCUSSION THREAD

**Discussion by machacekch** [bitcraze/discussions#1404] [Q&A] [answered]
I ordered an AI deck 1.1 last week and received one with a date of 2023-09-01 on the bag. Turns out it's Rev.C1 so I'll have to go through the hassle of [updating the GAP8 bootloader](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#gap8-bootloader) to make OTA flashing work according to [this info from July 2022](https://github.com/orgs/bitcraze/discussions/159#discussioncomment-3267460). In August 2022 [this user claimed](https://github.com/orgs/bitcraze/discussions/173#discussion-4281959) to have received a Rev.D deck.

Why is the old Rev.C1 AI deck still being shipped in June 2024? Getting it to work requires additional hardware.

**Comment by knmcguire** [comment]
Hi! Did you buy them from our store (store.bitcraze.io) or from a local retailer?

**Reply by tobbeanton** [reply]
Even though there is a bootloader on most of the AI-decks a debugger is mandatory to do anything useful. e.g. see [this](https://github.com/orgs/bitcraze/discussions/1048) discussion.

**Reply by machacekch** [reply, marked as answer]
> Did you buy them from our store (store.bitcraze.io) or from a local retailer?

I got the AI deck from the bitcraze online store.

> a debugger is mandatory to do anything useful

I'm not committed to the platform (yet) and still evaluating it. Checking the image quality and wifi speed by running the streaming example without jumping through hoops would have been _useful to me_.

---

In the end I did purchase the [ARM-USB-TINY-H programmer](https://www.olimex.com/Products/ARM/JTAG/ARM-USB-TINY-H/), the [ARM-JTAG-20-10 adapter](https://www.olimex.com/Products/ARM/JTAG/ARM-JTAG-20-10/) (which unlike the bitcraze debug adapter comes with the required ribbon cable that has the 0.05" connectors) and a USB A<->B cable. Flashing the bootloader, then flashing the streaming example via `cfloader` over the air worked as explained in the docs.

So I'm no longer blocked by this. I'll leave it up to you to decide if this thread should be closed now.

**Reply by knmcguire** [reply]
Alright, thanks for letting us know! I'll just mark your latest update as an answer then
